docs/units/equivalencies.rst

```diff
@@ -529,6 +529,46 @@ for more information.
 
 .. EXAMPLE END
 
+Magnetic Flux Density Magnetic Field Strength Equivalency
```

Comment:
Why is "Magnetic" repeated twice? Is this intentional?

Reply:
```suggestion
Magnetic Flux Density and Magnetic Field Strength Equivalency
```
This was intentional but I understand it is not as readable! I have made a suggestion, alternatively could always drop the second magnetic as it is implied but I was erring on the side of caution.